src/app/models/notification/commented.rb

```diff
@@ -0,0 +1,5 @@
+module Notification
+  class Commented < Notification
+    has_many :functions, as: :notifiable, dependent: :destroy
```

Comment:
how is one notification has many functions? 
I think this line isn't needed at all, the relation is already polymorphic